New package: AlgebraicOptimization v0.0.1

JuliaRegistrator opened the pull request:
- Registering package: AlgebraicOptimization
- Repository: https://github.com/AlgebraicJulia/AlgebraicOptimization.jl
- Created by: @tylerhanks
- Version: v0.0.1
- Commit: 907bfbf4b12db1c3b03ce71e564f4a5208ac34a3
- Reviewed by: @tylerhanks
- Reference: https://github.com/AlgebraicJulia/AlgebraicOptimization.jl/commit/907bfbf4b12db1c3b03ce71e564f4a5208ac34a3#commitcomment-143734688
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6e69ff4c0f9de1dfffca5c4cb3117f5f4e8c79fe06b52678a47af1c432015fff5eaa68e23e9306567a1a3197a46c445a702cce348fd8aacd343fb23677474bb56b1afffeeace7f2da73c1b8e1d5d3b953d19c83f2480c204da20b6b5ffc987dded9902ede39ba0e6ca66b9bf8fe3aed8294ba5faaca33dd129c922b30a2d50540a6fc99b6336f022fe51cb48eef7e29c5b1f780900cff7e2c775cbbd7308231a5afd71fbb3c6cb8187310df3850d9a76a1f0cb23d4ee3fc85dc34c41f31d0b196 -->

tylerhanks commented:
> This looks very interesting, but it still seems to be lacking documentation at the moment. Maybe it's just that the badge links in the README aren't pointing to the right location. Alternatively, could you add some usage examples to the README while a more complete documentation is pending?

Yes more complete documentation is pending. I will add some usage examples to the README.

JuliaRegistrator committed:
New package: AlgebraicOptimization v0.0.1

UUID: a72ceada-00ec-4ad9-90d3-37b40eaed052
Repo: https://github.com/AlgebraicJulia/AlgebraicOptimization.jl.git
Tree: c1985e54b9543a8f0fc65c4d2c1388d294e18024

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

goerz commented:
[noblock] That looks much better now! Would you like me to unblock the registration so that it auto-merges, or is this PR too out of date now?

tylerhanks commented:
> [noblock] That looks much better now! Would you like me to unblock the registration so that it auto-merges, or is this PR too out of date now?

Hey @goerz , I think I want this unblocked for auto-merging. Just to make sure, by re-triggering the julia registrator on the commit which added the usage example, the released version will be of that commit right? Thanks!

goerz commented:
[noblock] You don’t have to retrigger the registration (and it’s probably best not to). The only thing holding back the auto-merge are your own blocking comment. You have to edit *all* comments to contain the word `[noblock]`
